lib/components/Form/form.stories.js

```diff
@@ -90,7 +90,7 @@ stories.add('Form Group', () => (
 stories.add('Auto Complete', () => (
   <UI>
     <Component>
-      <AutoComplete onChange={() => { }} items={[{ text: 'item value 1' }, { text: 'item value 2' }]} searchValue="item" />
+      <AutoComplete onChange={() => { }} items={[{ text: 'item value 1' }, { text: 'item value 2' }]} searchValue="Item" />
```

Comment:
Should we change this one to be upper case or the declaration to be lower case?

Reply:
Either way works as a test